File: 3625-stone-removal-game/3625-stone-removal-game.js
/**
 * @param {number} n
 * @return {boolean}
 */
var canAliceWin = function(n) {
    for (let i = 10; i >= 0; i--) {
        if (n < i)
            return i & 1 != 0;
        n -= i;
    }
    return false;
};

File: 3625-stone-removal-game/3625-stone-removal-game.py
class Solution(object):
    def canAliceWin(self, n):
        """
        :type n: int
        :rtype: bool
        """
        for i in range(10, -1, -1):
            if n < i:
                return i & 1 != 0
            n -= i
        return False